src/main/java/org/opentripplanner/updater/trip/TimetableSnapshotSource.java

```diff
@@ -1117,12 +1165,33 @@ private TripPattern getPatternForTripId(FeedScopedId tripId) {
     return transitEditorService.getPatternForTrip(trip);
   }
 
-  private static void debug(FeedScopedId id, String message, Object... params) {
-    debug(id.getFeedId(), id.getId(), message, params);
+  private static void debug(
+    FeedScopedId id,
+    LocalDate serviceDate,
+    String message,
+    Object... params
+  ) {
+    debug(id.getFeedId(), id.getId(), serviceDate, message, params);
+  }
+
+  private static void debug(String feedId, String message, Object... params) {
+    debug(feedId, null, null, message, params);
   }
 
-  private static void debug(String feedId, String tripId, String message, Object... params) {
-    String m = "[feedId: %s, tripId: %s] %s".formatted(feedId, tripId, message);
+  private static void debug(
```

Comment:
This method should have java doc explaining why this debug logging is ok - it goes against the general strategy in OTP to not use debug logging - It is however ok to do it for the interface of a component. The JavaDoc should explain what kind of problems this solves: "This debug logging help us ...."

Reply:
JavaDoc on one of these method is enough (the super method).

Reply:
Javadoc has been added -- I've modified all the logging statements to use a common method so that all the messages are similar.